### DISCUSSION TITLE
Packaging shiori for winget, or at least as MSIX on Windows

### DISCUSSION THREAD

**Discussion by torculus** [go-shiori/shiori#781] [Feature requests]
So [`winget`](https://github.com/microsoft/winget-pkgs) is a popular package manager for Windows, as well as the official package manager for Windows 11. Winget can also handle details such as setting the path for command-line tools (like `shiori`).

The only prerequisite is that `winget` requires programs to come with their own installers (`shiori` currently just comes as a `.tar.gz` with the `.exe` in it). So `shiori` would have to be packaged with an installer like the [MSIX Packaging Tool](https://apps.microsoft.com/detail/9N5LW3JBCXKF?rtc=1&hl=en-us&gl=US). But even if we don't upload it to `winget`, packaging it as a MSIX installer would dramatically improve the installation experience for Windows users

**Comment by fmartingr** [comment]
I can't advocate for windows as it's not my main OS, but since to interact with Shiori you need to use the CLI in some way, how dramatic would be having an installer?

I only see it that way if:
- We could add a system daemon to windows to auto-launch the server on system startup
- Add some kind of `shiori.local` reverse proxy to easy access the server.

There could be an alternative route, which is what syncthing does which is a tray daemon that has some quality of life for windows users.

I'm just making comments, but I don't have much time to investigate this. Potentially with #780 a revisit on the CI/CD pipelines would be needed. I can try to see if there's an easy way for us to provide a MSIX installer while I am on that, but can't make any promises.
